launcher/minecraft/auth/MinecraftAccount.cpp

```diff
@@ -181,8 +181,10 @@ void MinecraftAccount::authFailed(QString reason)
             data.validity_ = Validity::None;
             emit changed();
         } break;
+        case AccountTaskState::STATE_WORKING: {
+            data.accountState = AccountState::Unchecked;
+        } break;
         case AccountTaskState::STATE_CREATED:
```

Comment:
Wouldn't created fall under Same unchecked category?

Reply:
The other cases are not reachable - this accidentally was not accounted for!